src/annotation.ts

```diff
@@ -154,13 +154,11 @@ export class Annotation extends Modifier {
   // Render text beside the note.
   draw(): void {
     const ctx = this.checkContext();
-
-    if (!this.note) {
-      throw new RuntimeError('NoNoteForAnnotation', "Can't draw text annotation without an attached note.");
-    }
-
+    this.checkAttachedNote();
```

Comment:
As the other check methods, this one could return the note and avoid the need fo the `getNote` below.

Reply:
Sounds good. It does seem odd to have a method named "check" that is actually a getter. :-D But as long as we are consistent with it, hehe.

Reply:
I just followed the convention that was already there with `checkContext`. I produced a `checkStave`...